css/tabstrip.css

```diff
@@ -67,10 +74,10 @@ main.showtabstrip iframes {
   margin-right: 0;
 }
 
-.tab:hover {
+.tab:hover > .tab-thumb {
```

Comment:
Direct descendent are actually supposed to be faster https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Writing_efficient_CSS. Though if we JIT CSS like Chrome, it shouldn't really matter.

At some point, I would like to rejigger the CSS so we use naming conventions instead of chaining selectors.